CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+---
```

Comment:
Not sure why this change log was generated -- i don't see any commits for cognito identity provider, ec2, nor aws kms around the 2/16/2020 timeframe.  Looked in both develop & master branches.. hmmm

Reply:
Hmm, i already merged in eb94180bd4ca62edd75c38911587b01698ef5a2c, which references both of the services noted in this commit, and we haven't done a release, yet, so i am going to disregard this PR.